### PR TITLE
NEW Adds the option of not displaying linked objects in the left header on the Sponge invoice template.

### PR DESCRIPTION
**Pull request by nat-itopie**
NEW Adds the option of not displaying linked objects in the left header on the Sponge invoice template.

### CODE REVIEW

**Comment by nat-itopie** [comment]
Sorry for the 2nd commit, it's a mistake, I don't know how to undo it.

**Comment by eldy** [comment]
> Sorry for the 2nd commit, it's a mistake, I don't know how to undo it.

Go on the lical branch you did the development on, and do a git revert of the commit you don't want, then just repush. It shoul update the pr.

**Comment by eldy** [comment]
> Sorry for the 2nd commit, it's a mistake, I don't know how to undo it.

The second commit just means you update your development branch before submitting pr. It is a good thing.

However, when you did your pr, did you check the checkbox "pr can be modified by others" ?
If not, can you redo a pull then push of your pr branch ? Or redo the pr by checking the checkbox so i can do it myself.

File: htdocs/langs/fr_FR/admin.lang [hunk 1]
```diff
@@ -2397,6 +2397,8 @@ INVOICE_SHOW_SHIPPING_ADDRESS=Afficher l'adresse de livraison
 INVOICE_SHOW_SHIPPING_ADDRESSMore=Indication obligatoire dans certains pays (France, ...)
 SUPPLIER_PROPOSAL_ADD_BILLING_CONTACT=Afficher le contact de facturation sur devis
 SUPPLIER_PROPOSAL_ADD_BILLING_CONTACTMore=Par défaut le contact n'apparaît que pour la facturation
+INVOICE_HIDE_LINKED_OBJECT=Cacher les objets liés
```
Comment (eldy) [inline htdocs/langs/fr_FR/admin.lang:2400]: No need to update fr language file. Only en_US is required.

**Pull request closed without merging**